app/Model/Event.php

```diff
@@ -113,6 +113,20 @@ private function findEventsNearNow($beforeOrAfterNow, $limit) {
             die("Event.findEventsNearNow: invalid parameter: $beforeOrAfterNow");
         }
 
+        if ($series_id != NULL) {
+            if ($series_id > 0) {
+                $conditions['AND'][] = array(
+                    'Event.series_id =' => $series_id,
+                );
+            } else {
+                // Non-series events
+                // TODO - Adjust DB to NOT NULL?
```

Comment:
sorry, what was the context on this comment?

Reply:
Got it, thanks for the context! I'd prefer to not do that as this one field's null state would then be inconsistent from the rest of the DB.

Reply:
Sure, no problem, I've removed the comment.

I think I've dealt with all of your comments now, let me know if there's more.